modal/functions.py

```diff
@@ -496,6 +496,8 @@ def from_args(
         allow_background_volume_commits: Optional[bool] = None,
         block_network: bool = False,
         max_inputs: Optional[int] = None,
+        batch_max_num_inputs: Optional[int] = None,
+        batch_max_timeout_sec: Optional[float] = None,
```

Comment:
Super nit, but can we use `_secs` to be consistent with other fields?

```
$ git grep "_sec[s ]" modal_proto/
modal_proto/api.proto:  uint32 timeout_secs = 21;
modal_proto/api.proto:  uint32 task_idle_timeout_secs = 25;
modal_proto/api.proto:  optional uint32 timeout_secs = 6;
modal_proto/api.proto:  optional uint32 task_idle_timeout_secs = 7;
modal_proto/api.proto:  uint32 timeout_secs = 7;
```

Reply:
Also, while I'm being really annoying, "max timeout seconds" doesn't make sense IMO.

The concept of a "max" waiting period is already included in the "timeout" term. But reaching a "timeout" is usually bad / an error. So I would say something like `max_wait_secs` is a better name here.

Reply:
that makes sense! sounds good!